sky/serve/serve_utils.py

```diff
@@ -673,6 +697,8 @@ def format_service_table(service_records: List[Dict[str, Any]],
             replica_infos.append(replica)
 
         service_name = record['name']
+        version = (record['version']
+                   if 'version' in record else constants.INITIAL_VERSION)
```

Comment:
```suggestion
                   if 'version' in record else '-')
```
we should show `-` for those old services?